recipes/scikit-cycling/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "scikit-cycling" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "3acaed9ccfdfa34f43e75342bd86c18bb3ad4b940a24eedb78f674364fba1076" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-cycling/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - export CC=clang  # [osx]
+    - export CXX=clang++  # [osx]
+    - python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - toolchain
+    - cython
+    - numpy 1.9.*  # [not (win and py36)]
+    - numpy 1.11.*  # [win and py36]
+    - clangdev
+    - clangdev
+    - libcxx 4.0.*  # [osx]
+    - openmp 4.0.*  # [osx]
+  run:
+    - python
+    - scipy
+    - numpy >=1.9  # [not (win and py36)]
+    - numpy >=1.11  # [win and py36]
+    - pandas
+    - six
+    - python-fitparse
+    - joblib
+    - scikit-learn
+    - libcxx >=4.0  # [osx]
+    - openmp >=4.0  # [osx]
+
+test:
+  imports:
+    - skcycling
```

Comment:
Are there other packages that you can import? Or maybe other built-in tests that you can run to make sure everything is really working?

Reply:
I can run the test suit which take a couple of minutes. But we already have the continuous integration in the repo and we have `scikit-learn-wheels` which also test the same build that we upload in PyPi, so I found it as a bit overkill

Reply:
Because we are recompiling the native code, I think it would be good to rerun the tests.

Reply:
fair enough. I will also inspect linkage and object after install

Reply:
You don't have to run the whole test suite if you don't want to / it takes a while. But you should definitely run something that tests the OpenMP stuff actually worked.